packages/account/src/Styles/account.scss

```diff
@@ -2264,3 +2264,10 @@ $MIN_HEIGHT_FLOATING: calc(
         gap: 0.8rem;
     }
 }
+
+.employment-tin-section {
+    .employment-status-field {
+        margin-bottom: 3.2rem;
+        max-width: 400px;
```

Comment:
should we standardize and use `rem` here? 🤔

Reply:
Do u mean 40 rem?